salt/edx/prod.sls

```diff
@@ -34,11 +34,10 @@ configure_python_ppa_for_edx:
 install_os_packages:
   pkg.installed:
     - pkgs:
-        - git
+        - git: 2.9.2-0ppa1~ubuntu12.04.1
         - libmysqlclient-dev
-        - python-dev
```

Comment:
Shouldn't this be set to 2.7.3 - or is that not the version installed by Ansible?